src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const allValues = [];
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (allValues.includes(value)) {
```

Comment:
You can check 
```
if (!newRobot.value) {
  newRobot.value = key;
}
```

So you can avoid additional variable creation and additional `else` block.